packages/feature-toggles-client/src/to-feature-state.ts

```diff
@@ -0,0 +1,47 @@
+import { RawFeatureValues } from '@etrigan/feature-toggles'
+import { FeatureState } from '.'
+
+export function toFeatureState(rawFeatures: RawFeatureValues): FeatureState {
+    return Object.keys(rawFeatures).reduce<FeatureState>((features, featureKey) => {
+        const rawValue = rawFeatures[featureKey]
+        features[featureKey] = {
+            // TODO The user should be able to specify this, just putting the shape in for now
+            canOverride: false,
+            value: {
+                raw: rawValue,
+                boolean() {
+                    if (rawValue === undefined) {
+                        throw new Error(`${featureKey} is undefined`)
+                    }
+                    if (
+                        rawValue === true ||
+                        rawValue === 'true' ||
+                        rawValue === '1' ||
+                        rawValue === 1
+                    ) {
+                        return true
+                    }
+                    if (
+                        rawValue === false ||
+                        rawValue === 'false' ||
+                        rawValue === '0' ||
+                        rawValue === 0
+                    ) {
+                        return false
+                    }
+                    throw new Error(`${featureKey} value is not a boolean: ${rawValue}`)
+                },
+                string() {
```

Comment:
Do you have an example of what this would be? Guessing this covers the use case like breaking news when we wanted to have data sent instead of just a boolean.

Reply:
Yeah,

Lets say we wanted to play around with different breach screens, we could create `default`, `sport`, `promo` then render based on the toggle.

Or `default`, `experiment-1` with a 50/50 split.

Another example could be nielsen, `default` | `new-implementation` | `disabled`

Just more flexible than boolean values